tests/datasource/fluent/test_metadatasource.py

```diff
@@ -66,21 +69,12 @@ def get_context(
             cls._context = DataContext(context_root_dir=context_root_dir)
 
         assert cls._context
-        if cls._context.root_directory:
-            # load config and add/instantiate Datasources & Assets
-            config_path = pathlib.Path(cls._context.root_directory) / _config_file
-            cls._config = GxConfig.parse_yaml(config_path)
-            for ds_name, datasource in cls._config.datasources.items():
-                logger.info(f"Loaded '{ds_name}' from config")
-                cls._context._add_fluent_datasource(datasource)
-                # TODO: add assets?
-
         return cls._context
 
     @validate_arguments
     def __init__(self, context_root_dir: Optional[DirectoryPath] = None) -> None:
         self.root_directory = context_root_dir
-        self._sources: _SourceFactories = _SourceFactories(self)
+        self._sources: _SourceFactories = _SourceFactories(self)  # type: ignore[arg-type]
```

Comment:
That's a weird one

Reply:
It complains that we are using this mock `DataContext` instead of `AbstractDataContext`.

Reply:
This is because `DataContext` is a fake and doesn't inherit from `AbstractDataContext` or implement the same interface. To fix this for real, we should change how we write this fake, eg inherit from `AbstractDataContext` or implement a standard interface.